cql2/standard/annex_ats_extended-basic-spatial-functions.adoc

```diff
@@ -0,0 +1,103 @@
+=== Conformance Class "Extended Basic Spatial Functions"
+
+:conf-class: extended-basic-spatial-functions
+[[conf_extended-basic-spatial-functions]]
+[cols="1,4a",width="90%"]
+|===
+2+|*Conformance Class*
+2+|http://www.opengis.net/spec/{standard}/{m_n}/conf/{conf-class}
+|Target type |Servers that evaluate filter expressions
+|Parameter |Filter Language: "CQL2 Text" or "CQL2 JSON"
+|Requirements class |<<rc_extended-basic-spatial-functions,Requirements Class "Extended Basic Spatial Functions">>
+|Dependency |<<conf_basic-cql2,Basic CQL2>>
+|===
+
+The term "geometry data type" is used for the following data types: Point, MultiPoint, LineString, MultiLineString, Polygon, MultiPolygon, or GeometryCollection.
+
+:conf-test: s_intersects
+==== Conformance Test {counter:test-id}
+[cols=">20h,<80a",width="100%"]
+|===
+|Test id: | /conf/{conf-class}/{conf-test}
+|Requirements: | <<req_{conf-class}_spatial-predicate,/req/{conf-class}/spatial-predicate>>, <<req_{conf-class}_spatial-functions,/req/{conf-class}/spatial-functions>>, <<req_{conf-class}_spatial-functions,/req/{conf-class}/spatial-data-types>>
+|Test purpose: | Test the S_INTERSECTS spatial comparison function with points, multi-points, line strings, multi-line string, polyons, mult-polygons, geometry collections and bounding boxes.
+|Test method: | 
+Given:
+
+* One or more data sources each with a list of queryables.
+* At least one queryable has a geometry data type.
+
+When:
+
+For each queryable `{queryable}` with a geometry data type, evaluate the following filter expressions
+
+* `S_INTERSECTS({queryable},BBOX(-180,-90,180,90))`
+* `S_INTERSECTS({queryable},POINT(7.02 49.92))`
+* `S_INTERSECTS({queryable},POINT(90 180))`
+* `S_INTERSECTS({queryable},BBOX(-180,-90,-90,90)) AND S_INTERSECTS({queryable},BBOX(90,-90,180,90))`
+
```

Comment:
The tests should also cover all other data types that must be supported (MultiPoint, LineString, MultiLineString, Polygon, MultiPolygon, GeometryCollection), not just BBox and Point.